includes/class-alg-wc-custom-order-statuses-core.php

```diff
@@ -83,7 +83,7 @@ public function __construct() {
 			}
 
 			// Emails.
-			if ( 'yes' === apply_filters( 'alg_orders_custom_statuses', 'no', 'value_emails' ) ) {
+			if ( 'yes' === get_option( 'alg_orders_custom_statuses_emails_enabled', 'no' ) ) {
 				add_action( 'woocommerce_order_status_changed', array( $this, 'send_email_on_order_status_changed' ), PHP_INT_MAX, 4 );
 			}
 
```

Comment:
What does this change in code do? The if condition I mean.

Reply:
This change is done as in lite we are not adding the filter :-  `alg_orders_custom_statuses` .So to make the global email function work this changes are done.